translate documentation for database engines (#26895)
ClickHouse/ClickHouse: docs/en/engines/table-engines/integrations/kafka.md

-   `kafka_row_delimiter` — Delimiter character, which ends the message.
-   `kafka_schema` — Parameter that must be used if the format requires a schema definition. For example, [Cap’n Proto](https://capnproto.org/) requires the path to the schema file and the name of the root `schema.capnp:Message` object.
-   `kafka_num_consumers` — The number of consumers per table. Default: `1`. Specify more consumers if the throughput of one consumer is insufficient. The total number of consumers should not exceed the number of partitions in the topic, since only one consumer can be assigned per partition.

> -   `kafka_max_block_size` - The maximum batch size (in messages) for poll (default: `max_block_size`, If `kafka_max_block_size` parameter setting, will use `max_block_size` and `kafka_max_block_size` Minimum value in size.).

Reviewer: ```suggestion
-   `kafka_max_block_size` - The maximum batch size (in messages) for poll (default: `max_block_size`, but not larger than current `max_block_size`.
```

Author: > Probably some files were added by accident?

Sorry, I'll revise it
